package-lock.json

```diff
@@ -1,5584 +1,8 @@
 {
   "name": "golpo13-backend",
   "version": "0.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Can you please revert this? And don't push any changes for pakage-lock.json 🙏

Btw is there any reason to use version 1?